### PR TITLE
Updated and corrected documentation.

### PR DESCRIPTION
**Pull request by Geraden07**
See issue #372 for description as to why these documentation changes are needed.

### CODE REVIEW

**Comment by jimklimov** [comment]
LGTM

File: docs/man/upscli_connect.txt [hunk 1]
```diff
@@ -24,7 +24,7 @@ connection, or `UPSCLI_CONN_REQSSL` to require a SSL connection.
 
 Introduced in version 2.7, an additional flag `UPSCLI_CONN_CERTVERIF`
 now exists to verify the signature offered during the SSL handshake.
-This flag should be used in conjuction with linkman:upscli_init[3]
+This flag should be used in conjunction with linkman:upscli_init[3]
```
Comment (jimklimov) [inline docs/man/upscli_connect.txt]: Nice, the first independent victim of Travis-based spellchecking! ;)

**Comment by jimklimov** [comment]
By the way, should any of this nuance be also referred to from some of the `conf/ups*.conf.sample(.in)` files, near the configuration comments for crypto setup?

File: docs/man/upscli_connect.txt (hunk 1 above)
Comment (Geraden07) [inline docs/man/upscli_connect.txt]: I even had that little bit of indignation that a lowly spell check broke my build, and how dare it? I was obviously just going to have to define some new words for it to pass but then, sure enough, an *actual* typo! Yikes. Shame on me. :-P

**Comment by Geraden07** [comment]
I can check the ups.conf.sample.in and the upsd.conf and upsmon.conf files for any further issues to be consistent with this, but it will have to be later in the week when I have some time. Also, I'll resolve the nut.dict conflict. I noticed it's been kept in alphabetical order so I'll merge and re-sort it.

**Comment by jimklimov** [comment]
Note that now there should be a "make spellcheck-sortdict" in the upstream
to facilitate the sorting (and uniq'ing and recounting) the words; it is
part of "make spellcheck-interactive".

2017-01-31 13:21 GMT+01:00 Steven B <notifications@github.com>:

**Comment by Geraden07** [comment]
I've spot checked the other documentation files mentioned in #372 and believe everything to be accurate with regards to capath and such.

I've pulled from upstream and merged the conflicted nut.dict file so I believe everything is good to go.

**Pull request merged**